src/test/java/org/arquillian/tests/pagetests/BlogPageTest.java

```diff
@@ -72,4 +72,21 @@ public void should_be_able_to_go_to_jacoco_blog_from_cloud_tag() {
                 .hasTitle()
                 .hasReleaseNotes();
     }
+
+    @Test
+    public void should_redirect_to_new_announcement_if_banner_is_present() throws Exception {
+        mainPage.menu()
+            .navigate().to("Blog");
+
+        blogPage.blogContent()
+            .verify()
+            .hasAnnouncementBanner(true);
```

Comment:
In case the first page of the blog will contain only the latest versions/announcements, then the test will fail

Reply:
Updated